quantum/keymap_extras/keymap_greek.h

```diff
@@ -114,7 +114,7 @@
 #define GR_PLUS S(GR_EQL)  // +
 // Row 2
 #define GR_COLN S(GR_SCLN) // :
-#define GR_DTON S(GR_FSIG) // ΅ (dead)
+#define GR_DIAT S(GR_FSIG) // ΅ (dead)
```

Comment:
Why this change? You suggested `DTON` in #8636.

Reply:
I realized that `DIAT` might make more sense since I found out that “tonos” means accent. So this would be similar to how in `RNGA` and `BRKP` the “modifier” word takes up the first three letters. Plus, now I think that `DIAT` pairs nicer with `DIAE` ¯\\\_(ツ)\_/¯